css/tablesort.css

```
@@ -0,0 +1,26 @@
+table[data-sortable-table] th[role=columnheader]:not(.no-sort) {
+    cursor: pointer;
+}
+
+table[data-sortable-table] th[role=columnheader]:not(.no-sort):after {
+    content: '▾';
+    float: right;
+    visibility: hidden;
+    opacity: 0;
+    -webkit-user-select: none;
+    user-select: none;
+}
+
+table[data-sortable-table] th[aria-sort=ascending]:not(.no-sort):after {
```

```suggestion
table[data-sortable-table] th[aria-sort=descending]:not(.no-sort):after {
```

Otherwise the arrows behave differently than in the original script.